templates/experiments/chat/components/message_rating.html

```diff
@@ -0,0 +1,20 @@
+<div class="flex flex-row gap-1" id="rating-{{message.id}}">
+    {% if message.rating %}
+        <div class="text-sm">{{ message.rating }}</div>
+    {% else %}
+        <button
+            hx-post="{% url 'experiments:rate_message' team_slug=team_slug message_id=message.id rating='👍' %}"
+            hx-swap="outerHTML"
+            hx-target="#rating-{{message.id}}"
+            class="text-sm p-1 rounded-md hover:bg-gray-50">
+            👍
```

Comment:
add in a title to indicate what the icon means-- in chatgpt they have "good response" which I think it pretty clear

Reply:
Good idea, I'll add